presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -150,7 +152,17 @@ public SystemSessionProperties(
                         DataSize.class,
                         memoryManagerConfig.getMaxQueryMemory(),
                         true,
-                        value -> DataSize.valueOf((String) value)));
+                        value -> DataSize.valueOf((String) value)),
+                booleanSessionProperty(
+                        COLUMNAR_PROCESSING,
+                        "Use columnar processing for dictionary encoded blocks",
```

Comment:
I don't think you want `for dictionary encoded blocks` in the description of this one